doc/metadata/README.md

```diff
@@ -0,0 +1,20 @@
+Verify in-toto's supply chain with in-toto
+===========================================
+
+**Overview**  
+This document describes how to use in-toto to verify the software supply chains for the PyPi source of in-toto v1.0.1.
```

Comment:
We should generalize these instructions as much as possible. We can take the tone of "Consider v1.0.1 as an example" or something like that.

Reply:
Additionally, we may want to either recommend starting from a clean clone or mention some common things to `--exclude`.